gateway/README.md

```diff
@@ -118,55 +118,46 @@ Content-Length: 2
 {}
 ```
 
-### Responses
+## Responses
 
-By default gRPC-Gateway translates non-error gRPC response into HTTP response
-with status code set to `200 - OK`.
+You may need to modify the HTTP response body returned by the gRPC gateway. For instance, the gRPC Gateway translates non-error gRPC responses into `200 - OK` HTTP responses, which might not suit your particular use case.
 
-A HTTP response returned from gRPC-Gateway does not comform REST API Syntax
-and has no `success` section.
+### Overwrite Default Response Forwarder
 
-In order to override this behavior gRPC-Gateway wiki recommends to overwrite
-`ForwardResponseMessage` and `ForwardResponseStream` functions correspondingly.
-See [this article](https://github.com/grpc-ecosystem/grpc-gateway/wiki/How-to-customize-your-gateway#replace-a-response-forwarder-per-method)
+By default, an HTTP response returned by the gRPC Gateway doesn't conform to the Infoblox REST API Syntax (e.g. it has no `success` section).
 
-#### How can I overwrite default Forwarders?
+To override this behavior, the gRPC Gateway documentation recommends overwriting `ForwardResponseMessage` and `ForwardResponseStream` functions correspondingly. See [this documentation](https://github.com/grpc-ecosystem/grpc-gateway/wiki/How-to-customize-your-gateway#replace-a-response-forwarder-per-method) for further information.
 
-```
+```go
 import (
-	"github.com/infobloxopen/atlas-app-toolkit/gw"
+	"github.com/infobloxopen/atlas-app-toolkit/gateway"
 )
 
 func init() {
 	forward_App_ListObjects_0 = gateway.ForwardResponseMessage
 }
 ```
 
-You can also refer [example app](https://github.com/github.com/infobloxopen/atlas-contacts-app/pb/contacts/contacts.overwrite.pb.gw.go)
+You can also refer [example app](https://github.com/infobloxopen/atlas-contacts-app/blob/master/pkg/pb/contacts.overwrite.pb.gw.go).
```

Comment:
This app is outdated but we can still refer it at least for this example. My concern is that some developers may consider using this app as an example. Do you think we should still refer old contacts app?

Reply:
Agreed. I'll remove this sentence.